src/config/dynamic-theme-fixes.config

```diff
@@ -269,6 +269,9 @@ CSS
 .markdown-body pre code {
     background-color: transparent !important;
 }
+.new-discussion-timeline .closed-banner {
+    border-bottom-color: rgb(23, 24, 28);
```

Comment:
The fix should look well in Light and Dark mode and filter values should be applied, for this purpose `${color}` templates are used, where `color` is the source color, usually white or black. Also `!important` should be added to prevent override by another rule.